src/IdentityManager/WebApi/Models/AuthResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IdentityManager.API.Models
+{
+    public class AuthResponse
+    {
+        public string AccessToken { get; set; }
```

Comment:
We should not return Id4 specific info (like AccessToken or RefreshToken ) from our endpoint. That will do Id4 enpoint itself.